src/Microsoft.AspNet.Mvc.Razor/Resources.resx

```diff
@@ -141,6 +141,12 @@
   <data name="MvcRazorCodeParser_OnlyOneModelStatementIsAllowed" xml:space="preserve">
     <value>Only one '{0}' statement is allowed in a file.</value>
   </data>
+  <data name="RazorPage_ThereIsNoActiveWritingScopeToEnd" xml:space="preserve">
+    <value>There is no active writing scope to end.</value>
+  </data>
+  <data name="RazorPage_YouCannotFlushWhileInAWritingScope" xml:space="preserve">
+    <value>You cannot flush while inside a writing scope.</value>
```

Comment:
@tedd - if you're having a problem, please open a new issue and include a description and some code that will help us reproduce the issue.